src/unicef_security/views.py

```diff
@@ -15,5 +15,9 @@ def get_context_data(self, **kwargs):
 
 class UNICEFLogoutView(RedirectView):
     def get_redirect_url(self, *args, **kwargs):
-        return f'https://{settings.TENANT_B2C_URL}/{settings.TENANT_ID}/{settings.POLICY}/oauth2/v2.0/' \
+
+        base_url = getattr(settings, 'TENANT_B2C_URL', 'login.microsoftonline.com')
```

Comment:
if these are needed for default values please add them to config file same as UNICEF_EMAIL. you can import django conf.settings there as well.

Reply:
not sure that you read my message correctly.